### PR TITLE
Alt text for the Microsoft logo

### PR DESCRIPTION
**Pull request by dhoehna**
## Summary of the pull request
Hyper-V review tab uses the Microsoft Logo.  Added Alt Text.

## References and relevant issues

https://task.ms/50083926

## Validation steps performed

Manually verified.

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated
- [ ] Telemetry [compliance tasks](https://aka.ms/devhome-telemetry) completed for added/updated events

Movie!

https://github.com/microsoft/devhome/assets/2517139/7c619b7d-0edf-45a2-9908-530e20b5e967

### CODE REVIEW

**Review by krschau — APPROVED** [review]

File: extensions/HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -357,4 +357,8 @@
     <value>Unable to start the virtual machine management service. The Hyper-V feature may have been recently enabled but a reboot has not been initiated. Please check that the service exists in the Windows Services snap-in if a reboot has already been done.</value>
     <comment>{Locked="{Hyper-V}", "{Windows}"} Error text to show the user that virtual machine management service can't be started by the extension</comment>
   </data>
+  <data name="MicrosoftLogoAltText" xml:space="preserve">
+    <value>Microsoft Logo</value>
```
Comment (bbonaby) [inline extensions/HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw:361]: Oh so, these images actually change based on the image you select. E.g if you select ubuntu, the ubuntu image will show up. My suggestion would be, to use the Image name as the first word then logo. So something like:

`{0} Logo`

Then in VMGalleryCreationAdaptiveCardSession.cs you could do 

```C#
{ "DiskImageAltText", _stringResource.GetLocalized("ImageLogoAltText", galleryImage.Name);},
```

Comment (dhoehna) [reply extensions/HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw]: Nice catch.  Changed.  Check the new video in the description.

File: extensions/HyperVExtension/src/HyperVExtension/Models/VMGalleryCreationAdaptiveCardSession.cs [hunk 2]
```diff
@@ -213,6 +213,7 @@ private async Task<ProviderOperationResult> GetForReviewFormAdaptiveCardAsync(st
 
             var galleryImage = _vMGalleryImageList.Images[inputForGalleryOperation.SelectedImageListIndex];
             var newEnvironmentNameLabel = _stringResource.GetLocalized("NameLabelForNewVirtualMachine", ":");
+            var microsoftLogoAltText = _stringResource.GetLocalized("MicrosoftLogoAltText");
```
Comment (jaholme) [inline extensions/HyperVExtension/src/HyperVExtension/Models/VMGalleryCreationAdaptiveCardSession.cs:216]: >MicrosoftLogoAltText [](http://example.com/codeflow?start=69&length=20)

I'm not seeing this string in the resw

Comment (jaholme) [reply extensions/HyperVExtension/src/HyperVExtension/Models/VMGalleryCreationAdaptiveCardSession.cs:216]: Ah this is just no longer used and should be deleted

File: extensions/HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw [hunk 3]
```diff
@@ -357,4 +357,8 @@
     <value>Unable to start the virtual machine management service. The Hyper-V feature may have been recently enabled but a reboot has not been initiated. Please check that the service exists in the Windows Services snap-in if a reboot has already been done.</value>
     <comment>{Locked="{Hyper-V}", "{Windows}"} Error text to show the user that virtual machine management service can't be started by the extension</comment>
   </data>
+  <data name="ImageLogoAltText" xml:space="preserve">
+    <value>{0} Logo</value>
+    <comment>Locked="{0}" The alt text for the Microsoft logo on the review tab. {0} is the name of the image.</comment>
```
Comment (bbonaby) [inline extensions/HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw:362]:
```Suggestion
    <comment>Locked="{0}" The alt text for the logo of the selected OS image on the review tab. {0} is the name of the image.</comment>
```

File: extensions/HyperVExtension/src/HyperVExtension/Models/VMGalleryCreationAdaptiveCardSession.cs (hunk 2 above)
Comment (dhoehna) [reply extensions/HyperVExtension/src/HyperVExtension/Models/VMGalleryCreationAdaptiveCardSession.cs]: I forgot to remove it.

**Review by AmelBawa-msft — APPROVED** [review]

**Pull request merged**